Muvr/MKExerciseSession+MRManagedExerciseSession.swift

```diff
@@ -0,0 +1,15 @@
+import MuvrKit
+
+extension MKExerciseSession {
```

Comment:
It's a bad idea to construct NSManagedObjects without using the NSEntityDescriptor.... functions. If you do, the CoreData attributes will not work.